src/api/one/scheme/tip/get-liked-tips.action/function.ts

```diff
@@ -0,0 +1,48 @@
+import { Action } from './interface'
+import { OneApiFunc } from '@/api/one/types'
+import { TipListResponse } from '@/database/models/tip'
+import { TipRepository } from '@/database/repository/tip-repository'
+import { oneApiResponse } from '@/api/one/utils'
+import prisma from '@/modules/prisma'
+import { prismaTimeMod } from '@/modules/time'
+
+const func: OneApiFunc<Action> = async (data) => {
+  const { authPayload, page } = data
+  const { userId } = authPayload
+  const pageSize = 10
+
+  let totalCount = 0
+
+  totalCount = await prisma.tip.count({
+    where: {
+      isRemoved: false,
+      tipLikes: {
+        some: {
+          userId,
+        },
+      },
+    },
+  })
+  const tipList = await TipRepository.findLiked(userId, pageSize, page)
+
+  const tips = tipList.map((item) => {
+    const response: TipListResponse = {
+      ...item,
+      tipLikes: item.tipLikes.length,
+      isLiked: true,
+    }
+    return response
+  })
+
+  const totalPage =
+    totalCount % 10 === 0 ? totalCount / 10 : Math.floor(totalCount / 10) + 1
```

Comment:
It should be better to use `pageSize` variable instead of literal 10 for a single source of truth.

Reply:
@jhaemin fixed